sdk/communication/Azure.Communication.CallingServer/tests/Infrastructure/CallingServerTestEnvironment.cs

```diff
@@ -15,13 +15,18 @@ public class CallingServerTestEnvironment : CommunicationTestEnvironment
         /// <summary>
         /// The phone number associated with the source.
         /// </summary>
-        public string SourcePhoneNumber => GetRecordedVariable(AzurePhoneNumber, options => options.IsSecret());
+        public string SourcePhoneNumber => GetRecordedVariable(AlternateCallerId, options => options.IsSecret());
```

Comment:
If we are going to change it to alternateCallerId, can we change the variable name as well?

Reply:
We prefer to using the SourcePhoneNumber in comparasion with TargetPhoneNumber

Reply:
Can you clarify why?

Reply:
We have target phone number, so here we have source phone number, considering later on, the source phone number could be a pstn number that is not from ACS resource, then we assign another number and differentiate it with the AlternateCallerId from ACS.